telemetry/definitions/commonDefinitions.json

```diff
@@ -2825,20 +2815,6 @@
                 }
             ]
         },
-        {
-            "name": "codeTransform_isDoubleClickedToTriggerUserModal",
-            "description": "The code transform button in the devtools to initiate the pre transform user modal window.",
-            "metadata": [
-                {
-                    "type": "codeTransformSessionId",
-                    "required": true
```

Comment:
by removing required:true types, are these metrics already deprecated in implementation?

Reply:
Yes none of these metrics are used right now.

Reply:
Double check: Does that exist for both Jetbrains and vscode? This will be a breaking change otherwise.

Reply:
Double checked and none of these metrics are currently used in JetBrains nor VS Code. As long as HIL didn't re-introduce any of these this change should be fine to make.